demo/api/src/products/generated/product.resolver.ts

```diff
@@ -200,6 +205,7 @@ export class ProductResolver {
 
     @Mutation(() => Product)
     @AffectedEntity(Product)
+    @RequiredPermission(["updateProduct"], { skipScopeCheck: true })
     async updateProductVisibility(
```

Comment:
updateProductVisibility could also be configured with an extra permission-option

Reply:
probably I can ignore this, as visibility-flag is deprecated in favor of status, as far as I know...